tree/ntuple/v7/src/RPageStorageDaos.cxx

```diff
@@ -40,6 +40,42 @@
 #include <regex>
 
 namespace {
+using AttributeKey_t = ROOT::Experimental::Detail::RDaosContainer::AttributeKey_t;
+using DistributionKey_t = ROOT::Experimental::Detail::RDaosContainer::DistributionKey_t;
+
+/// \brief RNTuple page-DAOS mappings
+enum EDaosMapping { kOClusterDColumn, kOUnique };
+
+struct RDaosKey {
+   daos_obj_id_t oid;
+   DistributionKey_t dkey;
+   AttributeKey_t akey;
+};
+
+/// \brief Some random distribution/attribute keys.
```

Comment:
Perhaps it would help if we briefly describe which ones are used for which mapping (or make that apparent in the name).